Gemfile

```diff
@@ -33,6 +36,8 @@ gem "paperclip"
 gem "aws-sdk"
 gem "sidekiq"
 
+gem "rgeo"
```

Comment:
This is already a dependency of `activerecord-postgis-adapter`, no need to include twice.